indico/modules/events/registration/forms.py

```diff
@@ -323,6 +323,11 @@ class TicketsForm(IndicoForm):
                                         description=_('Allow users to export their ticket to Google Wallet. '
                                                       'This currently does not support tickets for accompanying '
                                                       'persons.'))
+    ticket_apple_pass = BooleanField(_('Export to Apple Pass'), [HiddenUnless('tickets_enabled',
```

Comment:
It seems that Apple stopped using the name "Apple Pass" in favor of "Apple Wallet".

```suggestion
    ticket_apple_pass = BooleanField(_('Export to Apple Wallet'), [HiddenUnless('tickets_enabled',
```

Reply:
Done.
(Apple Wallet is just one of the App that will be able to open the .pkpass you are going to download: WalletPasses, YourWallet, Stocard - and soon the Google Wallet itself - are just few of the Android App that allows you to open a .pkpass file.